bin/RUV_normalisation.R

```diff
@@ -38,35 +47,42 @@ for (i in names(t)) {
   exp <- k$rnaseq
   eCounts <- assays(exp)$counts
   samples<-colnames(eCounts)
-  average.counts<-technical_replicate_average_gtex(exp,expAcc)
-  all <- cbind(all,average.counts)
+  if ( study_type == "tissues" ){
+    average.counts<-technical_replicate_average_gtex(exp,expAcc)
```

Comment:
Is this still only applicable to gtex only? I thought that the idea was that this was more generally applicable, but maybe I got that wrong.

Reply:
Same below with blueprints

Reply:
Good catch 👍.  Updated now with renaming the function `_gtex` to `_tissues`. Same for blueprint replacing with `_bp` to `_cell_types`